data/patterns.d/nethesis.ini

```
@@ -3,3 +3,9 @@ pattern = "e0e656([a-f0-9]{2})([a-f0-9]{2})([a-f0-9]{2})\.cfg"
 scopeid = "e0-e6-56-$1-$2-$3"
 template = "tmpl_phone"
 content_type = "text/plain; charset=utf-8"
+
+[nethesis-firmware]
+pattern = "nethesis_np_x([3,5])_hwv1_0\.txt"
```

Shorter subgroup-matching regexp.

```suggestion
pattern = "nethesis_np_x(3|5)_hwv1_0\.txt"
```

What about not using a regexp at all?

```
[nethesis-firmware-x3]
...

[nethesis-firmware-x5]
...
```